lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -348,6 +348,17 @@ function dosomething_helpers_ip_address() {
 }
 
 /**
+ * Utility function to check if current request is an api request.
+ *
+ * @return boolean
+ */
+function dosomething_helpers_is_api_request() {
+  $path = explode('/', request_path());
+
+  return in_array('api', $path);
```

Comment:
Should we compare against `$path[0]` here? Wondering if we'd ever want this to trigger for a page where the first part isn't `api/`... e.g. `www.dosomething.org/documentation/api` or something like that.